src/app/lib/utilities/srcSet/index.js

```diff
@@ -31,7 +31,7 @@ export const createSrcsets = ({
   imageResolutions = DEFAULT_RESOLUTIONS,
 }) => {
   if (originCode === 'pips') {
-    return { primarySrcset: null, fallbackSrcset: null };
+    return { primarySrcset: undefined, fallbackSrcset: undefined };
```

Comment:
why is this needed?

Reply:
Similar typing issue to the other `null` ones, whereby some TS components can supply a `string | undefined` rather than a `string | null`.

This shouldn't affect anything, but I agree is a weird oddity.